external/Kconfig

```diff
@@ -73,6 +73,7 @@ config ENABLE_IOTIVITY
 	default n
 	select LIBC_NETDB
 	select TIME_EXTENDED
+	select ARCH_STDARG_H
```

Comment:
ARCH_STDARG_H means that some architecture has a specific stdarg.h inside arch folder and will use it instead of TizenRT's stdarg.h. Do you want to use it? We have two boards, sidk_s5jt200 and artik053. Both doesn't have it.